File: async-scala-client/src/main/scala/io/swagger/client/model/PurchaseCalculatedTax.scala
package io.swagger.client.model

import org.joda.time.DateTime
import java.util.UUID


case class PurchaseCalculatedTax (
  taxByType: Option[PurchaseTaxByType],
tax: Option[Double],  // Sum of tax type not VAT (Value Added Tax)
details: Option[List[PurchaseTaxByTypeDetail]]
)

File: javascript-closure-angular-client/API/Client/PurchaseCalculatedTax.js
goog.provide('API.Client.PurchaseCalculatedTax');

/**
 * @record
 */
API.Client.PurchaseCalculatedTax = function() {}

/**
 * @type {!API.Client.PurchaseTaxByType}
 * @export
 */
API.Client.PurchaseCalculatedTax.prototype.taxByType;

/**
 * Sum of tax type not VAT (Value Added Tax)
 * @type {!number}
 * @export
 */
API.Client.PurchaseCalculatedTax.prototype.tax;

/**
 * @type {!Array<!API.Client.PurchaseTaxByTypeDetail>}
 * @export
 */
API.Client.PurchaseCalculatedTax.prototype.details;

